php/common/reports.php

```diff
@@ -101,13 +107,14 @@
                 MAX(done) AS done
             FROM Torrents
             WHERE error = 0
+                AND client_id NOT IN (' . $in . ')
```

Comment:
Bad indentation.

```suggestion
            AND client_id NOT IN (' . $in . ')
```

Reply:
We have any code conventions for sql code?
This is ok
```
        WHERE
            Torrents.info_hash IS NOT NULL
            AND Topics.ss = ?
            AND Topics.se / Topics.qt <= 10
```

and this is not?
```

            WHERE error = 0
                AND client_id NOT IN (' . $in . ')```

Reply:
IMO, this will be resolved in future branch.

Reply:
Ah, I got your point, @Evgen-BioNik. Then please disregard my comment. Also @Orhideous is right.